market_nextdoor_api/views.py

```diff
@@ -341,3 +344,35 @@ def update_preorder(preorder, data):
     preorder_data.save()
     return Response(preorder_data.data)
   return Response(status=status.HTTP_400_BAD_REQUEST)
+
+#USDA Market locations apikey=rTS5CqcxKA
+def get_market_locations(request, zipcode):
+  USDA_API_KEY = os.environ.get('USDA_API_KEY')
+  BASE_URL = "https://www.usdalocalfoodportal.com/api/farmersmarket/"
+  request_url = f'{BASE_URL}?apikey={USDA_API_KEY}&zip={zipcode}&radius=50'
+  head = {
+        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36'
+    }
```

Comment:
Great find here so that we can utilize the API. Thank you for doing the extra research to resolve the issue.